src/stylesheets/print.scss

```diff
@@ -21,9 +52,24 @@
     border-bottom: solid 5px black;
   }
 
-  h1 {
-    border-bottom: solid 5px #009bc3;
-    padding-bottom: 1rem;
-    margin-bottom: 3rem;
+  .easi-responsive-tabs__navigation {
```

Comment:
The styles from here to the rest of the file should probably go live in the component where they originate.